src/seedu/addressbook/AddressBook.java

```diff
@@ -461,7 +461,7 @@ private static String getMessageForSuccessfulAddPerson(String[] addedPerson) {
      * @return feedback display message for the operation result
      */
     private static String executeFindPersons(String commandArgs) {
-        final Set<String> keywords = extractKeywordsFromFindPersonArgs(commandArgs);
+        final Set<String> keywords = extractKeywordsFromFindPersonArgs(commandArgs.toLowerCase());
```

Comment:
Good job. Remember to update your test cases and user guide because there are modifications to the program behavior. Finally, just some nitpicking, please name your commits meaningfully. Instead of "Enhance the code", you may wish to include what you have done specifically: "Make the find command case insensitive".